src/main/java/org/apereo/openequella/adminconsole/service/JarService.java

```diff
@@ -136,7 +136,13 @@ public void downloadJar(String jarName) {
 	public int executeJar(String jarName, String mainClass, String... jvmArgs) {
 		final File binJar = getBinJarFile(jarName);
 		final List<String> command = new ArrayList<>();
-		command.add("java");
+		// Use the bundled JRE to run admin console
+		final String os = System.getProperty("os.name");
```

Comment:
Hmm, actually, you can't run this from the debugger anymore.  I will try to think of a solution